test/index.test.js

```diff
@@ -88,4 +88,37 @@ describe('percySnapshot', () => {
       '[percy] Error: failure\n'
     ]);
   });
+
+  it('works in standalone mode', async () => {
+    browser = null;
+
+    delete require.cache[require.resolve('..')];
+    percySnapshot = require('..');
```

Comment:
This is "polluting" the root level `percySnapshot`. The more isolated approach would be to define `percySnapshot` at the suite level and re-require it in `beforeEach`. Or we can leave the root level variable alone and redeclare it locally here within this scope. We can also utilize the sdk-util test helper, `sdk.rerequire()`, which uses [exactly this pattern](https://github.com/percy/cli/blob/master/packages/sdk-utils/test/helper.js#L40) that you have here.

```suggestion
    let percySnapshot = sdk.rerequire('..');
```